index.js

```diff
@@ -0,0 +1,145 @@
+const execFile = require('child_process').execFile;
+
+const helmBinary = process.env.HELM_BIN;
+const args = process.argv;
+
+/**
+ * Prints the extension's usage
+ */
+function printUsage() {
+  const usage = "Converts helm's output to json format.\n"
+    + 'Works for commands: install, status\n'
+    + '\n'
+    + 'Example usage:\n'
+    + '  helm json install stable/rabbitmq\n'
+    + '  helm json status my-release-name\n';
+  process.stdout.write(usage);
+}
+
+// The expected arguments are:
+// ['node', 'js file', 'command', 'chart / release name']
+if (args.length < 4) {
+  printUsage();
+  process.exit(0);
+}
+
+/**
+ * Builds a json object from a string array of resources.
+ * format example:
+ * resourcesArr = ['==> v1beta1/StatefulSet \n
+                    NAME   DESIRED  CURRENT  AGE \n
+                    fun-elk-mariadb  1        1        2s',
+                   '==> v1/Pod(related) \n
+                    NAME   READY  STATUS   RESTARTS  AGE \n
+                    fun-elk-wordpress-665ff69d4b-9kg8x  0/1    Pending  0  2s \n
+                    fun-elk-mariadb-0    0/1    Pending  0   2s']
+ */
+function ConvertToObject(resourcesArr) {
+  const json = [];
+  resourcesArr.forEach((element) => {
+    const lines = element.split('\n');
+    let name;
+    const resources = [];
```

Comment:
put constants together and keep a spacing between declarations and method calls (here and everywhere HAE)